workers/php/types/php-event-worker-task.d.ts

```diff
@@ -0,0 +1,17 @@
+import { EventWorkerTask } from '../../../lib/types/event-worker-task';
+import { DecodedEventData } from 'hawk.types';
+
+/**
+ * Describe a context passed from Php Catcher
+ */
+interface PhpEventPayload extends DecodedEventData {}
```

Comment:
pull the new version of the `hawk.types` package , please. There should be a generic with Addons. Something like that:

```ts
interface PhpEventPayload extends DecodedEventData<PhpAddons> {}
```